test/HyperliquidInterchainTokenService.js

```diff
@@ -33,30 +33,23 @@ describe('Hyperliquid Interchain Token Service', () => {
     describe('Hyperliquid Interchain Token Service Update Token Deployer', () => {
         beforeEach(async () => {
             const salt = getRandomBytes32();
-            tokenId = await tokenFactory.linkedTokenId(wallet.address, salt);
-
-            testToken = await ethers.getContractFactory('TestHyperliquidInterchainToken', wallet);
-            testToken = await testToken.deploy('TestToken', 'TEST', 18, service.address, tokenId);
-
-            await tokenFactory.registerCustomToken(salt, testToken.address, 1, wallet.address);
+            await tokenFactory.deployInterchainToken(salt, 'TestToken', 'TEST', 18, 1000000, wallet.address);
```

Comment:
```suggestion
            await tokenFactory.deployInterchainToken(salt, 'TestToken', 'TEST', 18, 1000000, wallet.address).then((tx) => tx.wait());
```
tx.waits are missing when broadcasting txs (double check all tests?). Needed when running tests against live networks